lib/utils/getServerlessConfigFile.test.js

```diff
@@ -146,6 +148,81 @@ describe('#getServerlessConfigFile()', () => {
     ).to.be.rejectedWith('serverless.js must export plain object');
   });
 
+  it('should return the file content if a serverless.ts file found', () => {
```

Comment:
For tests we should use `runServerless` util, as documented here: https://github.com/serverless/serverless/blob/master/tests/README.md#unit-tests

It's the only reliable way to confirm that it works. We may prepare some basic fixture that involves `serverless.ts` config (but with no real TS internals, to not impose TS dependency on test engine)

Reply:
I added a fixture and added a test case.  Let me know if you think there's more that I should be testing